docs/root/intro/arch_overview/advanced/attributes.rst

```diff
@@ -152,6 +152,7 @@ The following attributes are available once the upstream connection is establish
    upstream.sha256_peer_certificate_digest, string, SHA256 digest of the peer certificate in the upstream TLS connection if present
    upstream.local_address, string, The local address of the upstream connection
    upstream.transport_failure_reason, string, The upstream transport failure reason e.g. certificate validation failed
+   upstream.request_attempt_count, int, The count of upstream request attempts. A value of ‘0’ indicates that the request was never attempted upstream
```

Comment:
int->uint. In CEL, uint is a distinct type, so you have to write literals as `3u` and convert explicitly if you want uint.

Reply:
@kyessenov Thanks for pointing it out. Should we keep this as `uint` or switch it to string?

Reply:
Don't think it's that important. `int` or `uint` are fine, but please be consistent throughout.

Reply:
Got it, Thanks. I switched everything to unit